Jenkinsfile

```diff
@@ -39,7 +40,7 @@ pipeline {
                     withKubeConfig([credentialsId: 'jenkins-rancher-service-account-kubeconfig-test']) {
                         wrap([$class: 'BuildUser']) {
                             sh "pipenv clean"
-                            sh "pipenv install --ignore-pipfile --skip-lock --site-packages --index https://test.pypi.org/simple/ 'mpyl==$CHANGE_ID.*'"
+                            sh "pipenv install --ignore-pipfile --skip-lock --site-packages --index 'mpyl==${params.MPYL_RELEASE}'"
```

Comment:
Did you check what happens when you pass a normal version of mpyl instead of a test version?

Reply:
Good point. I've added a version check in the `select_version` function. It is only possible to install test version with the `--version` flag. You think it's necessary to be able to install normal versions as well?